doc/specifications/misc/query_types.md

```diff
@@ -202,14 +100,37 @@ custom QueryType. Example: `AcmeBundle\Ez\QueryType\LatestContentQueryType`.
 
 #### Using a service tag
 If the proposed convention doesn't work for you, QueryTypes can be manually tagged in the service declaration:
+
 ```yaml
 acme.query.latest_content:
     class: AcmeBundle\Query\LatestContent
     tags:
         - {name: ezpublish.query_type}
 ```
 
-The effect is exactly the same than registering by convention.
+The effect is exactly the same than registering by convention. Defining a QueryType as a service is required
+if the class has custom dependencies.
+
+##### QueryType name override
+
+> Added in eZ Platform 1.7
+
+You may specify an 'alias' tag attribute that will be used to register the QueryType. It allows you to use the same
+class, with different arguments, as different QueryTypes:
+
+```yaml
+acme.query.latest_articles:
+    class: AcmeBundle\Query\LatestContent
+    arguments: ['article']
+    tags:
+        - {name: ezpublish.query_type, alias: latest_articles}
+
+acme.query.latest_links:
+    class: AcmeBundle\Query\LatestContent
```

Comment:
> afaik no, I just saw it in the removed text. but there are still ways to use from twig and yml right? and they will be able to use alias I assume? that would be great to have example on.

Okay. I have actually removed it since it had been declined, and took the opportunity to move the spec outside of "proposed".